classes/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Deck } from './Deck'
+export { default as Player } from './Player'
+export { default as Grid } from './Grid'
```

Comment:
I didn't know you could implicitly import and then export in one line like this, very nice 👍